libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -33,49 +35,73 @@ namespace dev
 namespace solidity
 {
 
-void ASTJsonConverter::addJsonNode(
+ASTJsonConverter::ASTJsonConverter(bool _legacy, map<string, unsigned> _sourceIndices):
+	m_legacy(_legacy),
+	m_sourceIndices(_sourceIndices)
+{
+}
+
+
+void ASTJsonConverter::setJsonNode(
 	ASTNode const& _node,
 	string const& _nodeName,
```

Comment:
I dont understand what you are reffering to here?  Or is it obsolete now that we add the nodes in the correct order?